socs/agents/pysmurf_controller/agent.py

```diff
@@ -1156,7 +1156,18 @@ def restart_rssi(self, session, params):
             # if the system is locked up, spawning a SmurfControl instance will hang
             epics.caput(f"smurf_server_s{self.slot}:AMCc:RestartRssi", 1)
 
-            return True, "RestartRssi sent"
+            # check the system has recovered
+            slot = epics.caget(
+                f"smurf_server_s{self.slot}:AMCc:FpgaTopLevel:AmcCarrierCore:AmcCarrierBsi:SlotNumber"
+            )
+            session.data.update({
+                "fields": {"rssi_responsive": slot is not None},
+                "last_updated": time.time()
+            })
```

Comment:
Great, can you add an example to the docstring for this structure? Here's an example from another operation:
https://github.com/simonsobs/socs/blob/269330b275017407254134366c3c6b1b1daa0e51/socs/agents/pysmurf_controller/agent.py#L423-L432